.github/workflows/nginx-otel-module-check.yml

```diff
@@ -24,7 +24,8 @@ jobs:
       - name: Build module
         working-directory: build
         run: |
-          cmake -DNGX_OTEL_NGINX_BUILD_DIR=${PWD}/../nginx/objs ..
+          cmake -DNGX_OTEL_NGINX_BUILD_DIR=${PWD}/../nginx/objs \
+          -DNGX_OTEL_DEV=ON ..
```

Comment:
Let's add some indent here or even better align it with `-DNGX_OTEL_...` option from above line.